frontend/src/pages/LoginPage.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="row">
+    <q-card class="col-0 col-sm-5 bg-primary xs-hide">
```

Comment:
Two cards used in the login page, looks like two cards are stacked next to each other and there is a small gap inbetween.

Recommendation:

1.  Can we use just one "q-card" and create a divider with solid border inbetween instead?
2.  Aora's tag lines are conflicting in many places, in title it says Aora -> Ask, discover, decide, in login page it says "Ahead of the rest, always" in the readme it says "Access, organise, retrieve, archive". Shall we make it consistent?

Reply:
1- i know when you tested it on a small device the left card will disappear. 

**Ahead of the rest, always.**  is the correct one. i did not change the other one because their are not for this issue.
you create an issue for this and the documentation because also the Read-me is not completely correct.